src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,96 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private Object[] array = new Object[10];
+    private int size = 1;
```

Comment:
The initial size is set to 1, which is incorrect. It should be initialized to 0 since the list is empty initially.